scripts/install-dev-tools.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
```

Comment:
I don't think we should include this script in the repo. It doesn't allow for enough flexibility in python setups, vifrtual environment usage, etc. We should instead just document these packages as requirements in the README